src/librustc_mir/interpret/eval_context.rs

```diff
@@ -628,10 +659,7 @@ impl<'mir, 'tcx, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
         if cur_unwinding {
             // Follow the unwind edge.
             let unwind = next_block.expect("Encounted StackPopCleanup::None when unwinding!");
-            let next_frame = self.frame_mut();
```

Comment:
The way next_frame is used in this function makes me a bit unhappy, but I can't come up with a better scheme on the spot and it's preexisting anyway. So.. OK I guess